rpc/src/v1/impls/eth.rs

```diff
@@ -801,6 +801,36 @@ impl<C, SN: ?Sized, S: ?Sized, M, EM, T: StateInfo + 'static> Eth for EthClient<
 		}
 	}
 
+	fn submit_work_detail(&self, nonce: RpcH64, pow_hash: RpcH256, mix_hash: RpcH256) -> Result<SubmitDetailResult> {
```

Comment:
And just a small grumble -- we can refactor L806 to L813 to a helper function to avoid duplicate codes with [submit_work](https://github.com/paritytech/parity-ethereum/blob/34ea01b4804ab4b1020b097425d505e6af287fe9/rpc/src/v1/impls/eth.rs#L784).

Reply:
thanks for your advice. I have modified it.